stagecraft/settings/common.py

```diff
@@ -115,3 +116,8 @@ def load_databases_from_environment():
 
 STATIC_URL = '/static/'
 STATIC_ROOT = abspath(pjoin(BASE_DIR, 'assets'))
+
+DOGSLOW_LOG_TO_FILE = False
+DOGSLOW_TIMER = 1
```

Comment:
Gunicorn worker timeout is 2 seconds. Ideally I would want less than half a second to return a cold response, but this is starting point.